### PR TITLE
chore: auto pass changes to fullscreen

### PR DESCRIPTION
**Pull request by abdelaziz-mahdy**
removes the need to call .update, set state or rebuilding the widget in general will pass the changes to the full screen widget

### CODE REVIEW

File: media_kit_video/lib/src/video/video_texture.dart [hunk 1]
```diff
@@ -229,6 +229,45 @@ class VideoState extends State<Video> with WidgetsBindingObserver {
     super.didChangeDependencies();
   }
 
+  @override
+  void didUpdateWidget(covariant Video oldWidget) {
```
Comment (moffatman) [inline media_kit_video/lib/src/video/video_texture.dart:233]: I think covariant should be removed

File: media_kit_video/lib/src/video/video_web.dart [hunk 2]
```diff
@@ -200,6 +200,45 @@ class VideoState extends State<Video> with WidgetsBindingObserver {
     );
   }
 
+  @override
+  void didUpdateWidget(covariant Video oldWidget) {
```
Comment (moffatman) [inline media_kit_video/lib/src/video/video_web.dart:204]: remove covariant here too

**Review by moffatman — APPROVED** [review]
just a minor thing

**Comment by abdelaziz-mahdy** [comment]
done.

**Pull request merged**